packages/shared/forks/ReactFeatureFlags.testing.www.js

```diff
@@ -61,7 +61,7 @@ export const enableSyncDefaultUpdates = true;
 export const allowConcurrentByDefault = true;
 export const enablePersistentOffscreenHostContainer = false;
 
-export const consoleManagedByDevToolsDuringStrictMode = true;
+export const consoleManagedByDevToolsDuringStrictMode = false;
```

Comment:
I'd expect this to be false because the "testing" builds of ReactDOM (which we currently don't actually use) would run in an environment that probably doesn't have DevTools, so it's not clear to me that this makes sense there